site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,409 @@
+
+This also fixes an issue where FMVs in Detective Pikachu would run faster than what the frame rate allowed for.
+
+{{< figure src="tomodachi.png"
+    title="Make sure to take your hayfever medicine, Frank!" >}}
+
+### service/apt: Implement Deliver Args ([#5611](https://github.com/citra-emu/citra/pull/5611)) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+This partially fixes an issue where some games would just show black and white screens when launched. By implementing Deliver Args, a way to pass arguments from the caller to the callee, as well as `PrepareToDoApplicationJump` and `JumpToApplication` APT services. This allows the game to crash with a fatal error instead of just hanging on a black screen. 
+
+This is a step in the right direction for getting these games, such as Sega 3D Classics Collection, to boot correctly on Citra in the future.
+
+### gl_rasterizer_cache: Remove all fully invalid surfaces from the cache ([#5710](https://github.com/citra-emu/citra/pull/5710)) by [BreadFish64](https://github.com/BreadFish64)
+
+The dead weight of fully invalid surfaces being held in the rasterizer cache impacted many cache management functions and texture lookup times, which in turn impacted performance on some games, such as Pilotwings.
+
+By removing all fully invalidated surfaces and implementing a map of recycled textures from destroyed surfaces to avoid driver overhead from allocations, performance on poorly behaved games has been greatly increased.
+
+{{< figure src="pilotwings.png"
+    title="A great boost in performance for problematic games!" >}}
+
+### Add svcGetSystemInfo 0x20000 to get Citra information ([#5955](https://github.com/citra-emu/citra/pull/5955)) by [PabloMK7](https://github.com/PabloMK7)
+
+`svcGetSystemInfo` is used by 3DS applications to gather the information of a system, such as whether someone is playing on a 3DS or 2DS, and to communicate this information back to the application. Homebrew apps also use this for the same purposes. 
+As homebrew apps had no way of telling whether they were running on hardware or on Citra, this implements a way for those aforementioned homebrew apps to finally tell if they are being run on Citra or not!
+
+### Allow MemoryRef to hold a past-the-end offset ([#6141](https://github.com/citra-emu/citra/pull/6141)) by [vitor-k](https://github.com/vitor-k)
+
+Previously, Citra did not correctly understand any copies or operations being held at the end of the emulated VRAM, which would cause these operations and copies to get completely ignored. This would cause numerous errors in the emulation of games, including some graphics being cut out entirely on HarmoKnight. By allowing MemoryRef to correctly understand these values, they will no longer be ignored and can be executed correctly.
```

Comment:
```suggestion
Previously, Citra did not correctly understand any copies or operations being held at the end of the emulated VRAM, which would cause them to get completely ignored. This would cause numerous errors in the emulation of games, including some graphics being cut out entirely on HarmoKnight. By allowing MemoryRef to correctly understand these values, they will no longer be ignored and can be executed correctly.
```